litentry/core/stf-task/receiver/src/lib.rs

```diff
@@ -106,7 +109,7 @@ impl<
 		Ok(())
 	}
 
-	pub fn create_verify_identity_trusted_call(
+	pub fn create_identity_verify_trusted_call(
```

Comment:
The intention was to create `verify_identity` trusted call, but I couldn't add quotes in name